test/TestCliRolePaths.py

```diff
@@ -90,12 +73,30 @@ def test_run_playbook(self):
         env = os.environ.copy()
         env['ANSIBLE_ROLES_PATH'] = os.path.dirname(cwd)
 
-        result = self.run_ansible_lint(cwd=cwd, role_path=role_path, env=env)
+        result = run_ansible_lint(cwd=cwd, role_path=role_path, env=env)
         self.assertIn('Use shell only when shell functionality is required', str(result))
 
     def test_run_role_name_invalid(self):
         cwd = self.local_test_dir
         role_path = 'roles/invalid-name'
 
-        result = self.run_ansible_lint(cwd=cwd, role_path=role_path)
+        result = run_ansible_lint(cwd=cwd, role_path=role_path)
         assert '106 Role name invalid-name does not match' in str(result)
+
+
+def test_run_playbook_github():
+    """Call ansible-lint simulating GitHub Actions environment."""
+    cwd = os.path.dirname(os.path.realpath(__file__))
+    role_path = '../examples/example.yml'
+
+    env = {
+        "GITHUB_ACTIONS": "true",
+        "GITHUB_WORKFLOW": "foo"
+    }
+
+    result = run_ansible_lint(cwd=cwd, role_path=role_path, env=env)
+    expected = (
+        '::error file=../examples/example.yml,line=47::[E101] '
```

Comment:
It is bad if the formatter really emits paths like this, with `../`. They should be resolved and relative to the git repo root. Otherwise, GHA won't match the annotations with files in the repo.

Reply:
Not really true. By design ansilble lint emits relative paths relative to CWD when command was called. Same behavior can be seen from other linters too. The "git" repo part is defined for all use-cases.

This path is like this because we call it from inside `./test`, but we can easily avoid it. Still, is not a bug.

Reply:
It is a bug because it breaks the main purpose of the feature.